FreeTDS 1.2 introduces CS_SEC_NETWORKAUTH but does not have CS_SEC_CHANBIND,
CS_SEC_CONFIDENTIALITY, CS_SEC_INTEGRITY, etc

diff --git a/src/cmd_connect.c b/src/cmd_connect.c
index 2956050..c81b3dd 100644
--- a/src/cmd_connect.c
+++ b/src/cmd_connect.c
@@ -1875,7 +1875,7 @@ static CS_RETCODE SetNetAuth (conn, principal, keytab_file, secmech, req_options
     CS_CHAR       *req_options;
 {
 
-#if defined(CS_SEC_NETWORKAUTH)
+#if defined(CS_SEC_NETWORKAUTH) && defined(CS_SEC_CHANBIND) && defined(CS_SEC_CONFIDENTIALITY) && defined(CS_SEC_INTEGRITY)
 
     CS_CHAR buf[CS_MAX_CHAR+1];
     CS_INT  buflen;
@@ -2051,7 +2051,7 @@ static CS_RETCODE ShowNetAuthCredExp (conn, cmdname)
     CS_CONNECTION *conn;
     CS_CHAR       *cmdname;
 {
-#if defined(CS_SEC_NETWORKAUTH)
+#if defined(CS_SEC_NETWORKAUTH) && defined(CS_SEC_CREDTIMEOUT)
     CS_INT     CredTimeOut;
     CS_BOOL    NETWORKAUTH;
     char      *datetime;
